client/app/bundles/course/assessment/submission/constants.js

```diff
@@ -22,6 +22,13 @@ export const workflowStates = {
   Published: 'published',
 };
 
+export const saveStatus = {
+  Initial: 'initial',
```

Comment:
Rename this to None? I think initial here does not really signify anything and may confuse developer on what's the behaviour of initial.